servers/sunrpc/dump_prog.go

```diff
@@ -1,4 +1,13 @@
-package sunrpcserver
+package sunrpc
+
+const (
+	dumpProgNum     = 123451501
+	dumpProgVersion = 1
+
+	_ = iota
+	gfDumpDump
+	gfDumpPing
+)
```

Comment:
Something has changed here. I get this now:

```
2017-01-30 19:59:12.453864 I | Remote server has no such procedure: {ProgramNumber:123451501 ProgramVersion:1 ProcedureNumber:1}
INFO[0025] glusterfs client connected                    address=127.0.0.1:49149
2017-01-30 19:59:12.456761 I | Remote server has no such procedure: {ProgramNumber:123451501 ProgramVersion:1 ProcedureNumber:1}
INFO[0025] glusterfs client connected                    address=127.0.0.1:49148
```

I can take a look at this later.